docs/standards/net_standards.md

```diff
@@ -0,0 +1,60 @@
+# .NET Standards
+
+At Defra, we use .NET as a secondary technology stack, with **Node.js** being our primary stack. The use of .NET must be justified with an **evidence-based decision**. This document outlines the standards for using .NET within Defra, ensuring alignment with modern practices and Microsoft's best practices.
+
+## Key Principles
+
+1. **Primary Stack Preference**:
+   - **Node.js** is the preferred stack for all new projects.
+   - The decision to use .NET must be supported by clear evidence, such as:
+     - Integration with commodity products or libraries that are best suited .NET.
```

Comment:
Missing the word "to" - should say "...best suited to .NET."

Reply:
Fixed